websites/A/Animes Vision/dist/metadata.json

```diff
@@ -4,20 +4,23 @@
 		"name": "FelipeSazz",
 		"id": "326123612153053184"
 	},
-	"contributors": [{
-		"name": "RisingSunLight",
-		"id": "240521747852558347"
-	}],
+	"contributors": [
+		{
+			"name": "RisingSunLight",
+			"id": "240521747852558347"
+		}
+	],
 	"service": "Animes Vision",
 	"description": {
 		"en": "Animes Vision is one of the largest Brazilian anime sites.",
+		"fr": "Animes Vision est un des plus grand sites d'animés Brésilien.",
 		"pt_BR": "Animes Vision é um dos maiores sites brasileiros de anime.",
 		"ga_IE": "Tá Animes Vision ar cheann de na suíomhanna anime Brasaíle is mó.",
 		"nl": "Animes Vision is een van de grootste Braziliaanse anime sites.",
 		"vi_VN": "Animes Vision là một trong những trang anime lớn nhất Brazil."
 	},
 	"url": "animes.vision",
-	"version": "2.0.0",
+	"version": "2.1.0",
```

Comment:
```suggestion
	"version": "2.0.1",
```

Reply:
I was already a contributor so idk
Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8